docs/pages/Configuration.md

```diff
@@ -461,12 +461,21 @@ Unless otherwise noted in the table below, the UI5 Tooling modules are backward
 
 Version | UI5 CLI Release
 --- | ---
+**2.2** | v2.4.0+
 **2.1** | v2.2.0+
 **2.0** | v2.0.0+
 **1.1** | v1.13.0+
 **1.0** | v1.0.0+
 **0.1** | v0.0.1+
 
+### Specification Version 2.2
+
+**Features:**
+
+- TODO
```

Comment:
Likely marked by intention as an open issue?

Reply:
Indeed, I'll take care of it. Thanks!

Reply:
Done in 8cedb82